SmartPlaces.Facilities/lib/IngestionManager/src/AzureDigitalTwins/AdtGraphManager.cs

```diff
@@ -0,0 +1,382 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="AdtGraphManager.cs" company="Microsoft">
+//   Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace IngestionManager.AzureDigitalTwins
+{
+    using System;
+    using System.Collections.Concurrent;
+    using System.Text.Json;
+    using Azure;
+    using Azure.DigitalTwins.Core;
+    using Azure.Identity;
+    using IngestionManager.Extensions;
+    using IngestionManager.Interfaces;
+    using Microsoft.ApplicationInsights;
+    using Microsoft.ApplicationInsights.Metrics;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Extensions.Options;
+
+    public class AdtGraphManager<TOptions> : IOutputGraphManager
```

Comment:
Maybe since this is open source `AdtGraphManager` should change to `AzureDigitalTwinsGraphManager`?